01-Login/exec.ps1

```diff
@@ -1,2 +1,2 @@
-docker build -t auth0-spring-mvc-webapp .
-docker run -p 8080:8080 -it auth0-spring-mvc-webapp
+docker build -t auth0-samples/spring-mvc-00login:latest .
```

Comment:
In some repos you've used just the repo name for the tags. We should be consistent with that.